packages/flutter/lib/src/material/flat_button.dart

```diff
@@ -76,20 +78,30 @@ class FlatButton extends StatelessWidget {
   /// Defaults to a color derived from the [Theme].
   final Color disabledTextColor;
 
-  /// The color of the button, as printed on the [Material]. Defaults to null,
-  /// meaning that the color is automatically derived from the [Theme].
+  /// The primary color of the button, as printed on the [Material], while it
+  /// is in its default (unpressed, enabled) state.
   ///
-  /// Typically, a material design color will be used, as follows:
-  ///
-  /// ```dart
-  ///  new FlatButton(
-  ///    color: Colors.blue[500],
-  ///    onPressed: _handleTap,
-  ///    child: new Text('DEMO'),
-  ///  ),
-  /// ```
```

Comment:
ok. i'll add it back. I'd personally question the "Typically, a material design color will be used" statement but I didn't do the user study so I trust you :)